crates/ty_python_semantic/resources/mdtest/type_qualifiers/classvar.md

```diff
@@ -101,6 +101,19 @@ class C:
     x: ClassVar[int, str] = 1
 ```
 
+## Trailing comma creates a tuple
+
+A trailing comma in a subscript creates a single-element tuple. We need to handle this
+gracefully and emit a proper error rather than crashing (see ty#1793).
```

Comment:
this isn't rendered very well by GitHub right now: https://github.com/astral-sh/ruff/blob/charlie/sli/crates/ty_python_semantic/resources/mdtest/type_qualifiers/classvar.md#trailing-comma-creates-a-tuple

```suggestion
gracefully and emit a proper error rather than crashing (see [ty#1793](https://github.com/astral-sh/ty/issues/1793)).
```